### PR TITLE
Inventory new3

### PR DESCRIPTION
**Pull request by CarlNord01**

### CODE REVIEW

File: Dungeon Crawler/Assets/Scripts/Inventory/InventoryItemController.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InventoryItemController : MonoBehaviour
+{
+    private Item item;
+
+    public Item Item
```
Comment (Kiopp) [inline Dungeon Crawler/Assets/Scripts/Inventory/InventoryItemController.cs:10]: Item Item is not acceptable

File: Dungeon Crawler/Assets/Scripts/Inventory/InventoryManager.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InventoryManager : MonoBehaviour
+{
+    public static InventoryManager Instance;
```
Comment (Kiopp) [inline Dungeon Crawler/Assets/Scripts/Inventory/InventoryManager.cs:8]: a lot of public properties, should probably be private?

**Review by Kiopp — CHANGES_REQUESTED** [review]
Just a few minor changes and it's good

**Review by mowaab — APPROVED** [review]

**Comment by Kiopp** [comment]
closed #12

**Pull request merged**